### DISCUSSION TITLE
Using commit message body instead of summary

### DISCUSSION THREAD

**Discussion by wolfy1339** [semantic-release/semantic-release#2286] [Q&A]
Hi there!

I am wondering if there is a way to configure `semantic-release` to use the commit body instead of the commit message.

Sometimes I have PRs that contain multiple related changes, but i want the release notes to reflect that automatically.
An example of this would be octokit/webhooks#576

In this situation, only Squash + Merge is available, thus I cannot just simply merge or rebase to keep the commits and have them reflect in the release notes.

**Comment by gr2m** [comment]
I wish that was possible. Maybe it is with a custom replacement for the https://github.com/semantic-release/release-notes-generator and/or https://github.com/semantic-release/commit-analyzer plugins? I never found the time to look into that, but am curious if anyone else did.

What I do right now is editing the release notes after semantic-release created the release
